core/opds2_import.py

```diff
@@ -764,6 +823,8 @@ def _extract_publication_metadata(
         )
         # Audiobook duration
         duration = publication.metadata.duration
+        # Not all parsers support time_tracking
+        time_tracking = getattr(publication.metadata, "time_tracking", False)
```

Comment:
Since playtime tracking is really only meaningful for audiobooks, it might be a good idea to make sure that if the flag is True, we are working with an audiobook. If the item isn't an audiobook, maybe we log a warning and ignore the flag.